packages/glimmer-apollo/src/index.ts

```diff
@@ -11,3 +11,15 @@ export type {
   UseMutation,
   UseSubscription
 } from './-private/usables';
+
+export { useResource } from './-private/use-resource';
+export {
```

Comment:
Let's export this as type.
```suggestion
export type {
```